### ISSUE TITLE
Carpet should not be required

### ISSUE THREAD

**Issue by haykam821** [diced/ServerStats#4]
According to the documentation, Carpet is used to get ticks per second and milliseconds per tick. However, Carpet adds several more invasive features which should not be required just to use this mod.

**Comment by diced** [comment]
Yes, this could be done without Carpet, I just decided to use it since it was a quick solution to get ticks per second and milliseconds per tick. What invasive features does carpet add though?
